samples/HostedInAspNet.Client/Program.cs

```diff
@@ -9,7 +9,7 @@ public class Program
     {
         static void Main(string[] args)
         {
-            new BrowserRenderer().AddComponent<Home>("app");
+            // Not used
```

Comment:
Guessing this change isn't going to be merged in the end

Reply:
FYI for this part, I just copied what you had already. I'll include a separate BlazorServerSide sample in the next iteration